src/components/mobile/mobile-next-query-preview.vue

```diff
@@ -9,21 +9,21 @@
     "
     :suggestion="nextQuery"
   >
-    <i18n class="x-title3 x-next-query-preview__title" tag="span" path="nextQueryPreview.message">
+    <i18n class="x-text1 x-next-query-preview__title" tag="span" path="nextQueryPreview.message">
```

Comment:
I think this should be title3

```suggestion
    <i18n class="x-title3 x-next-query-preview__title" tag="span" path="nextQueryPreview.message">
```

Reply:
I think it should be `x-text1-lg`